lib/sles4sap_publiccloud.pm

```diff
@@ -349,8 +354,8 @@ sub check_takeover {
     my ($self) = @_;
     my $hostname = $self->{my_instance}->{instance_id};
     my $retry_count = 0;
-    my $fenced_hana_status = $self->is_hana_online();
-    die("Fenced database '$hostname' is not offline") if ($fenced_hana_status == 1);
+    die("Fenced database '$hostname' is not offline") if ($self->is_hana_database_offline());
```

Comment:
Reading this outside of the code, it seems to me like it says: "Fenced database _is not offline_ if the hana database _is offline_". Sounds confusing and not natural. Perhaps the message in the `die` needs to be updated. Also, there is no such thing as a fenced database; perhaps you mean fenced node?

Reply:
ok